controllers/state_of_the_world.go

```diff
@@ -63,15 +71,122 @@ func NewPolicyMachineryController(manager ctrlruntime.Manager, client *dynamic.D
 }
 
 func buildReconciler(client *dynamic.DynamicClient) controller.ReconcileFunc {
+	authorinoCrSubscription := &controller.Subscription{
+		ReconcileFunc: NewAuthorinoCrReconciler(client).Reconcile,
+		Events: []controller.ResourceEventMatcher{
+			{Kind: ptr.To(kuadrantv1beta1.KuadrantKind), EventType: ptr.To(controller.CreateEvent)},
+			{Kind: ptr.To(kuadrantv1beta1.AuthorinoKind), EventType: ptr.To(controller.DeleteEvent)},
```

Comment:
I guess we also want to reconcile the Authorino CR when its updated, no? Or are we going to allow modifications to this CR after it was created by the operator? (I think we should not BTW.)

Reply:
Fair enough. Let's stick with the current behaviour then and keep the discussion about leaking abstractions vs ownership fully granted to the Kuadrant CR outside of this PR.